docs/navigating-sot-agg.md

```diff
@@ -2,17 +2,16 @@
 
 The Source of Truth Aggregation feature uses several key components:
 
-* A single GraphQL query which aggregates device data.
+* A GraphQL query, per settings instance, which aggregates device data.
 * A facility to modify data with a "transposer" function.
 * Nautobot's config context feature and policy engine.
 * Nautobot's native git platform.
 
 ## GraphQL
 
 There is currently support to make an arbitrary GraphQL query that has "device_id" as a variable. It is likely best to use the GraphiQL interface to model
-your data, and then save that query to the configuration. The application configuration ensures the following two components.
+your data, and then save that query as the Saved Query object. The application configuration ensures the following component.
```

Comment:
There is actually a way to bypass this now, and we should document it. e.g. if one goes to the query itself and updates it, this validation will not happen. I think this is fine, just need to make operators aware, via documentation.

Reply:
Makes sense. I've added a note saying that the validation rule only applies when creating/updating the Settings object.